Drivers/seds/PioHwas/example/work/hwas/implem/default/SDL/Makefile

```diff
@@ -0,0 +1,20 @@
+# SDL
+OPENGEODE?=opengeode
```

Comment:
I think that SDL Hwas implementation could be removed.

Reply:
Ok I will delete the whole SDL folder

Reply:
Fixed. Folder deleted